src/stp/new_roles/PenaltyKeeper.cpp

```diff
@@ -20,7 +20,7 @@ PenaltyKeeper::PenaltyKeeper(std::string name) : Role(std::move(name)) {
 Status PenaltyKeeper::update(StpInfo const& info) noexcept {
     // Failure if the required data is not present
     if (!info.getBall() || !info.getRobot() || !info.getField()) {
-        RTT_WARNING("Required information missing in the tactic info");
+        RTT_WARNING("Required information missing in the tactic info")
```

Comment:
Huh? Every statement should be closed by a semicolon, right?

Reply:
It's a macro defined to already contain the `;`